jina/parsers/logging.py

```diff
@@ -0,0 +1,10 @@
+"""Extensive argparser module for logging"""
+
+
+def mixin_suppress_root_logging_parser(parser):
+    parser.add_argument(
+        '--suppress_root_logging',

```

Comment:
CLI args are separated by `-` instead of underscore. The python variable name will be available with underscores.

Reply:
I get your point and make the change.